### PR TITLE
[Admin] add current Sylius version checker for admin panel notification

### LINKED ISSUE #8108: Sylius/GUS broken comparison logic.
There is no comparison logic on Sylius side so actually notification widget will always display that new version of sylius is available.

### PR DESCRIPTION
**Pull request by bitbager**
| Q               | A
| --------------- | ---
| Bug fix?        | yes |
| New feature?    | no |
| BC breaks?      | no |
| Related tickets | fixes #8108 |
| License         | MIT |

### CODE REVIEW

**Comment by pamil** [comment]
Btw. looks like you don't have your Git email configured at Github :)

File: src/Sylius/Bundle/AdminBundle/Controller/NotificationController.php [hunk 1]
```diff
@@ -81,6 +81,8 @@ public function getVersionAction(Request $request)
             return JsonResponse::create('', JsonResponse::HTTP_NO_CONTENT);
         }
 
-        return JsonResponse::fromJsonString($hubResponse->getBody()->getContents());
+        $hubResponse = json_decode($hubResponse->getBody()->getContents(), true);
+
+        return new JsonResponse($hubResponse);
```
Comment (pamil) [inline src/Sylius/Bundle/AdminBundle/Controller/NotificationController.php]: What's the point of decoding JSON just to encode it again?

File: src/Sylius/Bundle/AdminBundle/Controller/NotificationController.php [hunk 2]
```diff
@@ -23,7 +23,7 @@
 /**
  * @author Jan Góralski <jan.goralski@lakion.com>
  */
-class NotificationController
+final class NotificationController
```
Comment (pamil) [inline src/Sylius/Bundle/AdminBundle/Controller/NotificationController.php]: It's a minor BC break so it's OK for it to be introduced but we'd still need some info in UPGRADE.md

File: src/Sylius/Bundle/AdminBundle/Controller/NotificationController.php (hunk 1 above)
Comment (bitbager) [inline src/Sylius/Bundle/AdminBundle/Controller/NotificationController.php]: JsonResponse::fromJsonString does not work with ajax success function. I had not enough time to look deeper into what causes this problem, that's why I just left the new JsonResponse instance call. Looks like the fromJsonString was not used correctly.

File: src/Sylius/Bundle/AdminBundle/Controller/NotificationController.php (hunk 2 above)
Comment (bitbager) [inline src/Sylius/Bundle/AdminBundle/Controller/NotificationController.php]: Ok, I will put an extra line into the UPGRADE.md file tomorrow.

File: src/Sylius/Bundle/AdminBundle/Controller/NotificationController.php (hunk 1 above)
Comment (pamil) [inline src/Sylius/Bundle/AdminBundle/Controller/NotificationController.php]: Looks like it's related to encoding options (Symfony uses `JSON_HEX_TAG | JSON_HEX_APOS | JSON_HEX_AMP | JSON_HEX_QUOT`).

**Pull request merged**